charts/thoras/tests/secrets_collector_test.yaml

```
@@ -8,20 +8,20 @@ tests:
       slackWebhookUrlSecretRefKey: "url"
     asserts:
       - equal:
-          path: $..spec.containers[1].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.name
+          path: $..spec.containers[2].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.name
           value: "frou-frou"
       - equal:
-          path: $..spec.containers[1].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.key
+          path: $..spec.containers[2].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.key
           value: "url"
 
   - it: Points all apps to default slack secret, if no existing secret provided provided
     templates:
       - collector/deployment.yaml
     asserts:
       - equal:
-          path: $..spec.containers[1].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.name
+          path: $..spec.containers[2].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.name
```

[sand] Instead of simply updating indices, I'd suggest making these tests more resilient by constructing a [jsonpath query](https://github.com/json-path/JsonPath) that fetches the expected container, like this:

```suggestion
          path: $..spec.containers[?(@.name == 'thoras-collector')].env[?(@.name == 'SLACK_WEBHOOK_URL')].valueFrom.secretKeyRef.name
```

---

Ditto for other instances in this file that use hard-coded indices.